samples/ExampleGallery/Shared/GlowTextCustomControl.cs

```diff
@@ -142,7 +142,10 @@ protected override Size MeasureOverride(Size availableSize)
             var layout = CreateTextLayout(device, availableSize);
             var bounds = layout.LayoutBounds;
 
-            return new Size(Math.Min(availableSize.Width, bounds.Width + ExpandAmount), Math.Min(availableSize.Height, bounds.Height + ExpandAmount));
+            Size desiredSize = new Size(Math.Min(availableSize.Width, bounds.Width + ExpandAmount), Math.Min(availableSize.Height, bounds.Height + ExpandAmount));
+            canvas?.Measure(desiredSize);
```

Comment:
The ?. operator was a new addition in C# 6.0, but Win2D still supports Visual Studio 2013 so I'm afraid you cannot use this :-(